internal/expr/expr_test.go

```diff
@@ -421,95 +421,95 @@ func (s *ExprSuite) TestPrepareErrors(c *C) {
 	}{{
 		query:       "SELECT (p.name, t.id) AS (&Address.id) FROM t",
 		prepareArgs: []any{Address{}},
-		err:         "cannot prepare expression: mismatched number of columns and targets in output expression: (p.name, t.id) AS (&Address.id)",
+		err:         `cannot prepare statement: mismatched number of columns and target types in output expression: (p.name, t.id) AS (&Address.id)`,
```

Comment:
Very nitpicky, but do these quote marks need to change? I don't see any double-quotes here to avoid (unlike elsewhere)

Reply:
Changed back haha